src/app/api.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ApiService {
+  constructor(private http: HttpClient) {}
+
+  ping$(): Observable<any> {
+    console.log("http://localhost:3001");
+    return this.http.get(`http://localhost:3001/api/external`);
```

Comment:
Should this be changes to port 5000 since the backend specifies 5000 as its server port?